src/richie/apps/core/templates/richie/icons.html

```diff
@@ -201,5 +201,10 @@
       <path stroke-linecap="round" stroke-linejoin="round" d="M19.5 14.25v-2.625a3.375 3.375 0 0 0-3.375-3.375h-1.5A1.125 1.125 0 0 1 13.5 7.125v-1.5a3.375 3.375 0 0 0-3.375-3.375H8.25m0 12.75h7.5m-7.5 3H12M10.5 2.25H5.625c-.621 0-1.125.504-1.125 1.125v17.25c0 .621.504 1.125 1.125 1.125h12.75c.621 0 1.125-.504 1.125-1.125V11.25a9 9 0 0 0-9-9Z" />
     </symbol>
     
+    <symbol id="icon-video-play" viewBox="0 0 85 85" xmlns="http://www.w3.org/2000/svg">
+      <path style="fill:currentColor;stroke-width:.135807" d="M83.362 117.436a9.427 9.427 0 0 1-9.426 9.426 9.427 9.427 0 0 1-9.427-9.426 9.427 9.427 0 0 1 9.427-9.427 9.427 9.427 0 0 1 9.426 9.427z" transform="matrix(4.17778 0 0 4.17778 -269.089 -450.822)"/>
+      <path d="M204.11 0C91.388 0 0 91.388 0 204.111c0 112.725 91.388 204.11 204.11 204.11 112.729 0 204.11-91.385 204.11-204.11C408.221 91.388 316.839 0 204.11 0Zm82.437 229.971-126.368 72.471c-17.003 9.75-30.781 1.763-30.781-17.834V140.012c0-19.602 13.777-27.575 30.781-17.827l126.368 72.466c17.004 9.752 17.004 25.566 0 35.32z" style="fill:#fff" transform="scale(.19444)"/>
+    </symbol>
```

Comment:
We try to use as most as possible the same viewbox (`0 0 24 24`)for all icons as it ensure size consistency between Icons. I reworked our icon to match to viewbox

```suggestion
    <symbol id="icon-video-play" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
        <circle cx="12" cy="12" r="12" fill="#fff"/>
        <path fill-rule="evenodd" clip-rule="evenodd" d="m9.417 17.781 7.43-4.26c1-.574 1-1.504 0-2.077l-7.43-4.26c-1-.574-1.81-.105-1.81 1.048v8.5c0 1.153.81 1.622 1.81 1.05Z" fill="currentColor"/>
    </symbol>
```

Reply:
Ok. Thanks!